01-indoor-floor-level-tracker/web-app/src/services/ServiceLocatorServer.ts

```diff
@@ -40,32 +37,26 @@ class ServiceLocatorServer {
     if (!this.dataProvider) {
       const projectID = IDBuilder.buildProjectID(Config.hubProjectUID);
       const fleetID = IDBuilder.buildFleetID(Config.hubFleetUID);
+      const notehubJsClient = NotehubJs.ApiClient.instance;
```

Comment:
I did my best to DRY up the code in terms of adding the Notehub JS instances and auth as high up in the code as I could without making the whole app aware of the implementation details of our data fetching and auth, but if you can think of a way to make it cleaner, please advise.